packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -818,6 +818,30 @@ nodeunitShim({
 
     test.done();
   },
+
+  'cluster arn is correct when cluster identifier has uppercase letters'(test: Test) {
+    const stack = testStack();
+    const clusterIdentifier = 'TestClusterIdentifier';
+    const vpc = ec2.Vpc.fromLookup(stack, 'VPC', { isDefault: true });
+    const cluster = new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_MYSQL,
+      vpc,
+      clusterIdentifier,
+    });
+
+    test.deepEqual(
+      stack.resolve(cluster.clusterArn),
```

Comment:
I don't love this way of testing. I'd rather use a `CfnOutput`, and assert that on the template. See example: https://github.com/aws/aws-cdk/blob/5d6233164611d69ac1bf5c73e1518eb14dbace8d/packages/%40aws-cdk/core/test/cross-environment-token.test.ts#L73-L84